src/components/editor/editor.tsx

```diff
@@ -74,6 +75,7 @@ export class Editor implements ComponentInterface {
   }
 
   componentDidLoad() {
+    if (!configStore.state.dontInjectHeadCss)
```

Comment:
Not sure about this. It does not arm but if no css is injected then end user will not see the placeholder 'Press "/" for plugins'. Any idea how we can solve it?

Also the condition, if condition there is, should only avoid the injection of the css not the resize listener.

Reply:
For me this css does not help, cause stylo is used inside of shadow dom. So we could inject the css to the parent of the stylo component? Or I inject myself (what I do at the moment).
But we also could leave this, cause the inject css does not harm

Reply:
Your contenteditable finds place within your shadowed tree or is it pass as a slot?

If slot, the css can work, if not the css should be added to the shadowed tree. Not sure how to solve this in an elegant way.

Being said, using a css is the current solution. As the library is in alpha version, it might change until finalized.

Reply:
we could add it to the "adoptedStylesheets" -> https://github.com/WICG/construct-stylesheets
but this is only supported by chrome at the moment

Reply:
no, my contenteditable is i another webcomponent, wich is not accessible via a slot

Reply:
why don't you silve the "press / ..." not also via a overlay, the same as you do it for the toolbar?

Reply:
> we could add it to the "adoptedStylesheets" 

i wish to use "adoptedStylesheets" but no, solution should be crossbrowser.

> why don't you silve the "press / ..." not also via a overlay

interesting idea but want to stick to the current ux

> my contenteditable is i another webcomponent

thanks for the feedback, gotcha

well I don't have idea at the moment i have to say 🤷‍♂️. if you find further solution, happy to discuss these

otherwise spontaneously i have to say we should probably drop this new configuration, let the css be attached to the dom head and add a note in the readme about usage in web components, at least for now.